DeskReserve/Repository/IRoomRepository.cs

```diff
@@ -0,0 +1,18 @@
+using DeskReserve.Data.DBContext.Entity;
+using DeskReserve.Domain;
+
+namespace DeskReserve.Repository
+{
+	public interface IRoomRepository
+	{
+		Task<IEnumerable<Room>> GetAllAsync();
+
+		Task<Room> GetOneAsync(Guid id);
+
+		Task<bool> UpdateOneAsync(Room room);
+
+		Task<bool> CreateOneAsync(Room room);
+
+		Task<bool> DeleteOneAsync(Guid id);
```

Comment:
Use simpler names Get, Update, Create, Delete, why add suffix Async when the return type is Task ?